src/django_idom/views.py

```diff
@@ -0,0 +1,7 @@
+from django.http import HttpRequest, HttpResponse
+from idom.config import IDOM_WED_MODULES_DIR
+
+
+def web_modules_file(request: HttpRequest, file: str) -> HttpResponse:
```

Comment:
Add `lru_cache` decorator here.

Reply:
generic `funcutils.cache` decorator would work fine here too, since we don't really want to ever limit the amount of modules being cached.